volmdlr/edges.py

```diff
@@ -333,6 +379,20 @@ def simplify(self):
         """Search another simplified edge that can represent the edge."""
         return self
 
+    def is_point_any_end(self, other_point, abs_tol: float = 1e-6):
```

Comment:
```suggestion
    def is_edge_extremity(self, other_point, abs_tol: float = 1e-6):
```

Reply:
I think is_edge_extremity or is_point_edge_extremity or  is_point_at_edge_extremity would be better name options for this method